util/install/debian.sh

```diff
@@ -15,8 +15,14 @@ _qmk_install() {
         build-essential clang-format diffutils gcc git unzip wget zip \
         python3-pip binutils-avr gcc-avr avr-libc binutils-arm-none-eabi \
         gcc-arm-none-eabi libnewlib-arm-none-eabi avrdude dfu-programmer \
-        dfu-util teensy-loader-cli libhidapi-hidraw0 libusb-dev \
-        picolibc-riscv64-unknown-elf gcc-riscv64-unknown-elf binutils-riscv64-unknown-elf
+        dfu-util teensy-loader-cli libhidapi-hidraw0 libusb-dev
 
-    python3 -m pip install --user -r $QMK_FIRMWARE_DIR/requirements.txt
+    # RISC-V toolchains with picolibc support are only available from Debain 11+ and Ubuntu 21.04+ onwards.
```

Comment:
```suggestion
    # RISC-V toolchains with picolibc support are only available from Debian 11+ and Ubuntu 21.04+ onwards.
```
Also it seems that the requested packages are actually available since Ubuntu 20.10 (but I have no idea whether they are actually installable and working). Not sure whether mentioning it really matters, because that version is no longer supported now.

Reply:
Done.